text/0045-nft-deposits-asset-hub.md

```diff
@@ -0,0 +1,246 @@
+# RFC-0045: Lowering NFT Deposits on Asset Hub
+
+|                 |                                                                               |
+| --------------- | ----------------------------------------------------------------------------- |
+| **Start Date**  | 2 November 2023                                                               |
+| **Description** | A proposal to reduce the minimum deposit required for collection creation on the Polkadot and Kusama Asset Hubs. |
+| **Authors**     | [Aurora Poppyseed](https://github.com/poppyseedDev), [Just_Luuuu](https://github.com/justLuuuu), [Viki Val](https://github.com/vikiival) |
+
+## Summary
+
+This RFC proposes changing the current deposit requirements on the Polkadot and Kusama Asset Hub for
+creating an NFT collection, minting an individual NFT, and lowering its corresponding metadata and
+attribute deposits. The objective is to lower the barrier to entry for NFT creators, fostering a
+more inclusive and vibrant ecosystem while maintaining network integrity and preventing spam.
+
+## Motivation
+
+The current deposit of 10 DOT for collection creation (along with 0.01 DOT for item deposit and 0.2
+DOT for metadata and attribute deposits) on the Polkadot Asset Hub and 0.1 KSM on Kusama Asset Hub
+presents a significant financial barrier for many NFT creators. By lowering the deposit
+requirements, we aim to encourage more NFT creators to participate in the Polkadot NFT ecosystem,
+thereby enriching the diversity and vibrancy of the community and its offerings.
+
+The initial introduction of a 10 DOT deposit was an arbitrary starting point that does not consider
+the actual storage footprint of an NFT collection. This proposal aims to adjust the deposit first to
+a value based on the `deposit` function, which calculates a deposit based on the number of keys
+introduced to storage and the size of corresponding values stored.
+
+Further, it suggests a direction for a future of calculating deposits variably based on adoption
+and/or market conditions. There is a discussion on tradeoffs of setting deposits too high or too
+low.
+
+### Requirements
+
+- Deposits SHOULD be derived from `deposit` function, adjusted by correspoding pricing mechansim.
```

Comment:
> The terms "MUST", "MUST NOT", "SHALL", "SHALL NOT", or "REQUIRED" mean that the requirement is fixed and must be adhered to by implementations. These statements should be limited to those required for interoperability and security.
> The terms "SHOULD", "RECOMMENDED", "SHOULD NOT", or "NOT RECOMMENDED" mean that there are only limited valid circumstances in which a requirement may be ignored.

I would lean toward "SHOULD" here because it's not like a message format or protocol that other systems depend on. There _may be_ cases in the future where some deposit item is not based on such a function, and it's certainly not a technical limitation that other parts of the runtime depend on.

But either one works for me, I think this is a case where the both are OK.

Reply:
Yea okay. Its not an implementation detail anyway.